### PR TITLE
Implement WebSocket Networking

### PR DESCRIPTION
**Pull request by TheSecondReal0**

### CODE REVIEW

File: src/Scripts/network.gd [hunk 1]
```diff
@@ -7,5 +7,67 @@ enum Connection {
 }
 
 var connection = Connection.LOCAL
-var host: String = ''
+var host: String = '' #not sure why this is needed or a string, server network ID will always be 1
```
Comment (monban) [inline src/Scripts/network.gd]: This was intended to be used as the hostname of the server. It is set to localhost when connected locally (see [PlayGame.gd line 17](https://github.com/opensuspect/opensuspect/blob/d8bc5b247e45ce3d7fb701c66429039bf1416e1d/src/Scripts/PlayGame.gd#L17)).

Comment (TheSecondReal0) [reply src/Scripts/network.gd]: I see, maybe change the name of that variable to be hostName?

Comment (monban) [reply src/Scripts/network.gd]: I'm good with that.

Comment (monban) [reply src/Scripts/network.gd]: Would you please go ahead an change the variable name and remove the comment, so we can merge? Thank you.

**Comment by tooManyArguments** [comment]
Also might have to rebase since we changed file / dir structure:


Conflicting files
src/Scripts/network.gd

(all is now lowercase)

**Review by daniel071 — APPROVED** [review]
Just tested it and it works great:
image

Consider reducing the console spam:
image

When someone disconnects, this error occurs:
image

**Pull request closed without merging**